src/main/resources/db/changelog/2024/09/22-01-changelog.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
how do you gen these change logs?

Reply:
I generated these change logs by liquibase plugin in intelliJ